### PR TITLE
Fix test failure in CRLP_RollupContact_TEST 

### PR DESCRIPTION
**Pull request by lmeerkatz**
This only fails on certain dates. Tricksy!

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/classes/CRLP_RollupContact_TEST.cls [hunk 1]
```diff
@@ -348,6 +348,9 @@ private class CRLP_RollupContact_TEST {
                 npe01__Do_Not_Automatically_Create_Payment__c = true,
                 Type = 'New'
         ));
+        if (tt == TestType.TestWithAlwaysRollupToPrimary) {
+            donationYears.add(orgDonationDate.year().format());
```
Comment (force2b) [inline src/classes/CRLP_RollupContact_TEST.cls]: This will need to use the same `UTIL_String.removeNonNumericCharacters()` method as used on line 318.

Comment (lmeerkatz) [reply src/classes/CRLP_RollupContact_TEST.cls]: Why can't I use the actual CloseDate value? I thought this was what you were suggesting in chat as a replacement for removeNonNumeric ... ?

Comment (force2b) [reply src/classes/CRLP_RollupContact_TEST.cls]: `orgDonationDate.year().format()` will return `2,019`.

Comment (lmeerkatz) [reply src/classes/CRLP_RollupContact_TEST.cls]: Ah, okay. I wonder how the test was passing with that in there!

Comment (force2b) [reply src/classes/CRLP_RollupContact_TEST.cls]: Line 318 uses the method to strip out the comma from the year before adding it to the `donationYears` set. Line 352 has to add the year in the same format.

**Pull request merged**